yb-voyager/src/srcdb/postgres.go

```diff
@@ -1154,6 +1155,7 @@ func (pg *PostgreSQL) GetMissingAssessMigrationPermissions() ([]string, error) {
 	}
 
 	if result != "" {
+		vars.PgssEnabledForAssessment = false
```

Comment:
As discussed, lets return this boolean from this function and then you can use it directly in cmd package. No need of vars package (at least for now)